pyxrf/core/tests/test_yaml_param_files.py

```diff
@@ -63,7 +63,7 @@ def _generate_parameter_set_schema():
                 "type": "array",
                 "additionalItems": False,
                 "uniqueItems": True,
-                "items": [
+                "prefixItems": [
```

Comment:
Would it work with the older `jsonschema` versions too? Or should there be a minimum pin for the package?

I am a little concerned that the version is not available on conda-forge yet (https://anaconda.org/conda-forge/jsonschema), where the latest version is v3.2.0 at the moment.

Reply:
It works on v3.2.0. I believe that `item` in this context is deprecated.